modules/examples/seed/client/modules/app/src/main/resources/application.conf

```diff
@@ -0,0 +1,3 @@
+name = "SeedClient"
+host = "localhost"
+port = 19683
```

Comment:
We probably want something like this instead of specifying a fixed port:
https://stackoverflow.com/questions/2675362/how-to-find-an-available-port